platform/ui/src/components/InputRange/InputRange.tsx

```diff
@@ -34,12 +35,14 @@ const InputRange: React.FC<{
   inputClassName,
   labelClassName,
   labelVariant,
+  showLabel = true,
 }) => {
   const [rangeValue, setRangeValue] = useState(value);
+  useEffect(() => setRangeValue(value), [value]);
```

Comment:
Allow for the value property to update the range value.

Reply:
can you put this comment on the code?